website/project/views/comment.py

```diff
@@ -190,6 +294,53 @@ def add_comment(**kwargs):
     }, http.CREATED
 
 
+def get_file_provider(page, root_target):
+    if page == 'files':
+        if root_target.provider == 'googledrive':
+            return 'Google Drive'
+        elif root_target.provider == 'osfstorage':
+            return 'OSF storage'
+        else:
+            return root_target.provider.title()
```

Comment:
I might have commented on this in a commit comment, but using `str.title` is unreliable; better to use a mapping of "storage name" -> human-readable name.